src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -25,8 +25,7 @@ public class ConvertToJsonCommand : PSCmdlet
         [AllowNull]
         public object InputObject { get; set; }
 
-        private int _depth = 2;
-        private const int maxDepthAllowed = 100;
+        private const int MaxDepthAllowed = 1000;
```

Comment:
I am curios, why increase the max depth allowed? Was there a scenario/use-case blocked by having a depth limited to 100?

Reply:
@ahsonkhan The cmdlet has Depth parameter with default 64 (This is my suggestion for discussion in the PR).
MaxDepthAllowed is only used to write an error message to users before doing serialization.